src/nomadic/backup/commands.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import subprocess
+from pathlib import Path
+
+import click
+
+from nomadic.util.dirs import produce_dir
+from nomadic.util.logging_config import LoggingFascade
+from nomadic.util.rsync import selective_rsync
+
+script_dir = Path(__file__).parent.resolve()
+log = logging.getLogger(script_dir.stem)
+
+
+@click.command(short_help="Backup sequence data folder to a local hard disk drive")
```

Comment:
I would keep in general, as this command is at the moment not limited to a local hard disk. 
Also it is backing up the whole workspace, right?
So maybe something like: 
```suggestion
@click.command(short_help="Backup whole workspace together with minknow sequencing data to a different folder, e.g. on a local hard disk drive")
```

Reply:
Hope this is clearer